templates/accounts/dashboard.html

```diff
@@ -31,6 +41,32 @@ <h5 class="col-12 col-md-8 align-self-center text-center" id="sidebar-info" >{{
             </a>
         </div>
         <div class="offset-1 offset-sm-2 col-11 col-sm-10">
+            {% if active == "Admin"%}
```

Comment:
here don't use the active thing, instead, say '{% if user.is_staff %}'. Also you don't need to have a sperate thing for superusers, since all superusers know the urls to those links... 
So this staff panels need only be implemented in frontend, please delete the things in views...

Reply:
The **active attribute** indicates which of the 4 tabs is activated and therefore shown in the dashboard. The idea is to allow the staff to have access to the normal user dashboard interface, for debug/testing or whatever reasons, while providing a staff tab to give them quick access to the URLs.

Reply:
ah here I understand, you are correct